client/src/locale/ru-RU.locale.ts

```diff
@@ -0,0 +1,134 @@
+import Locale from './locale'
+
+export default <Locale>{
+    home: {
+        title: 'Паксталл',
+        subtitle: 'AUR для Ubuntu',
+        cards: {
+            whyDifferent: {
+                title: 'Чем это отличается от любого другого пакетного менеджера?',
+                description:
+                    'Паксталл использует стабильную базу Ubuntu, но позволяет использовать ' +
+                    "bleeding-edge программы почти что без компромиссов, так что вам не " +
```

Comment:
Is there an idiom for bleeding-edge you could use?

Reply:
Any updates?

Reply:
sorry for late response, fixed it!